index.js

```diff
@@ -18,67 +18,69 @@ module.exports = function(bot){
 
   let scheduler = () => bot.Scheduler || Scheduler;
 
-  var room = process.env.HUBOT_JOB_CHANNEL;
+  var mainRoom = process.env.HUBOT_JOB_CHANNEL;
```

Comment:
No reason to be using var.  Can just be const since you won't ever be changing it.